tests/test_datetime_parsers.py

```
@@ -21,403 +21,375 @@
+    assert day_sec_from_string('1998-12-31 23:59:60.99', timesys=True) == \
+        (-366, 86400.99, 'UTC')
+    assert day_sec_from_string('1998-12-31 23:59:60', timesys=False) == \
+        (-366, 86400)
+    assert day_sec_from_string('1998-12-31 23:59:60.99', timesys=False) == \
+        (-366, 86400.99)
+    assert day_sec_type_from_string('1998-12-31 23:59:60') == \
+        (-366, 86400, 'UTC')
+    assert day_sec_type_from_string('1998-12-31 23:59:60.99') == \
+        (-366, 86400.99, 'UTC')
+
+    assert type(day_sec_from_string('1998-12-31 23:59:60')[0]) == int
```

Ruff (0.15.6)

[error] Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)